src/main/java/frc/robot/subsystems/drivetrain/commands/HolonomicDrive.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.*;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import org.techfire225.webapp.FireLog;
+
+public class HolonomicDrive extends CommandBase {
+
+    private final SwerveDrive swerveDrive;
+
+    public HolonomicDrive(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void initialize() {
+        Robot.gyro.reset();
+    }
+
+    @Override
+    public void execute() {
+        double forward = Utils.joystickDeadband(-OI.xbox.getY(GenericHID.Hand.kRight));
+        double strafe = Utils.joystickDeadband(-OI.xbox.getX(GenericHID.Hand.kRight));
+        double rotation = Utils.joystickDeadband(-OI.xbox.getX(GenericHID.Hand.kLeft));
+        if (forward != 0 || strafe != 0 || rotation != 0) {
```

Comment:
Why is this condition needed?

Reply:
If I remember correctly we had a problem where the robot would move even if the joystick values were zero. @orel-david is that correct?

Reply:
Sometimes when we released the joysticks the modules returned to the origin when they could just stop instead